[2025-02-12T22:41:27.959+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-12T22:41:27.976+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-04T23:39:00+00:00 [queued]>
[2025-02-12T22:41:27.984+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-04T23:39:00+00:00 [queued]>
[2025-02-12T22:41:27.985+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-12T22:41:28.012+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_and_store_crypto_data> on 2023-10-04 23:39:00+00:00
[2025-02-12T22:41:28.019+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=18032) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-12T22:41:28.019+0000] {standard_task_runner.py:72} INFO - Started process 18042 to run task
[2025-02-12T22:41:28.019+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'crypto_data_collection', 'fetch_and_store_crypto_data', 'scheduled__2023-10-04T23:39:00+00:00', '--job-id', '6289', '--raw', '--subdir', 'DAGS_FOLDER/crypto_dag.py', '--cfg-path', '/tmp/tmp438dhypw']
[2025-02-12T22:41:28.020+0000] {standard_task_runner.py:105} INFO - Job 6289: Subtask fetch_and_store_crypto_data
[2025-02-12T22:41:28.057+0000] {task_command.py:467} INFO - Running <TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-04T23:39:00+00:00 [running]> on host be7ba0dd3a47
[2025-02-12T22:41:28.136+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_data_collection' AIRFLOW_CTX_TASK_ID='fetch_and_store_crypto_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-04T23:39:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-04T23:39:00+00:00'
[2025-02-12T22:41:28.137+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-12T22:41:29.852+0000] {logging_mixin.py:190} INFO - Données récupérées : {'Response': 'Error', 'Message': 'You are over your rate limit please upgrade your account!', 'HasWarning': False, 'Type': 99, 'RateLimit': {'calls_made': {'second': 8, 'minute': 75, 'hour': 4877, 'day': 5771, 'month': 5771, 'total_calls': 5771}, 'max_calls': {'second': 20, 'minute': 300, 'hour': 3000, 'day': 7500, 'month': 11000}}, 'Data': {}}
[2025-02-12T22:41:29.854+0000] {logging_mixin.py:190} INFO - Erreur lors de l'insertion dans PostgreSQL : string indices must be integers, not 'str'
[2025-02-12T22:41:29.901+0000] {logging_mixin.py:190} INFO - Données uploadées dans MinIO.
[2025-02-12T22:41:29.901+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-12T22:41:29.907+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-12T22:41:29.907+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=crypto_data_collection, task_id=fetch_and_store_crypto_data, run_id=scheduled__2023-10-04T23:39:00+00:00, execution_date=20231004T233900, start_date=20250212T224127, end_date=20250212T224129
[2025-02-12T22:41:29.930+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-12T22:41:29.941+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-12T22:41:29.945+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
